lib/go-atscfg/parentdotconfig.go

```diff
@@ -1325,3 +1346,48 @@ func GetDSOrigins(dses map[int]tc.DeliveryServiceNullableV30) (map[int]*OriginUR
 	}
 	return dsOrigins, nil
 }
+
+func makeDSOrigins(dsses []tc.DeliveryServiceServer, dses []tc.DeliveryServiceNullableV30, servers []tc.ServerNullable) map[DeliveryServiceID]map[ServerID]struct{} {
```

Comment:
It isn't strictly necessary to do this filtering. But it will make it easier to change ORT to use the origins table or any other DSS alternative in the future.